openflow_input/standard-1.5

```diff
@@ -830,26 +830,55 @@ enum ofp_stats_reply_flags(wire_type=uint16_t, bitmask=True) {
     OFPSF_REPLY_MORE = 0x1,
 };
 
-/* Multipart Message Types */
+/* 
+ * Multipart Message Types 
+ *
+ * OF1.5 renames the OFPMP/ST for:
+ * flow (1) -> flow_stats (17)
+ * aggregate (2) -> aggregate_stats (2)
+ * table (3) -> table_stats (3)
+ * group (6) -> group_stats (6)
+ * meter (9) -> meter_stats (9)
+ * meter_config (10) -> meter_desc (10)
+ * 
+ * OF1.4 renamed:
+ * queue (5) -> queue_stats (5)
+ * which was retained as "queue" in standard-1.4.
+ *
+ * Using different names for the same message
+ * in Loxi causes duplicate enum entries that
+ * map to essentially the same underlying message.
+ * For simpliciy, we can retain the original, 
+ * OF1.3-, naming convention to reduce complexity.
+ * This is why OF1.4+ ofp_stats_type enums from 
+ * OF1.3- remain as OFPST_* and do not appear as
+ * OFPMP_*.
+ *
+ * Using the _STATS is certainly more explicit; 
+ * however, that would perhaps necessitate changing
+ * OF1.3- OFPMP/ST names, which would break existing
+ * applications that use OF1.3- OFPMP/ST names,
+ * such as Floodlight, ONOS, etc.
+ */
 enum ofp_stats_type(wire_type=uint16_t) {
     OFPST_DESC = 0,
     OFPST_FLOW_DESC = 1,
```

Comment:
Let's keep the same name for messages across OF versions and ignore renames made by the spec. While there will need to be some special casing for OF 1.5, most of the codepath (for example, dispatching the message in the switch) will be the same.